test/integration/parsers/page_item_test.rb

```diff
@@ -58,7 +58,7 @@ class PageItemIntegrationTest < ActiveSupport::TestCase
     data = m.as_json
     assert_equal 'item', data['type']
     assert_equal 'page', data['provider']
-    assert_match(/Hong Kong Free Press/, data['title'])
+    assert_match(/Hong Kong lawmakers/, data['title'])
```

Comment:
These changes are unrelated to the PR, but the page title seems to have changed, so the test was failing. (Or it could be a flaky test)

Reply:
Thanks! Even if we simplify the `assert_match` to just match `/Hong Kong/`, it's already good enough.